### ISSUE TITLE
Unknown devices (MX400?)

### ISSUE THREAD

**Issue by Elektron79** [emsesp/EMS-ESP32#2198]
### DESCRIPTION

I have a Buderus WLW186i Heatpump with the „new“ MX400 Module which is the successor of the well known MX300 device.
In the Device Overview I have two „unknown“ devices. I expect that at least one is the MX400.
How can I support to enable the project to support also the MX400 Module?

### REQUESTED INFORMATION

_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [ x] Searched the issue in [issues](https://github.com/emsesp/EMS-ESP32/issues)
- [ ] Searched the issue in [discussions](https://github.com/emsesp/EMS-ESP32/discussions)
- [ ] Searched the issue in the [docs](https://emsesp.org/Troubleshooting/)
- [ ] Searched the issue in the [chat](https://discord.gg/3J3GgnzpyT)
- [ ] Provide the System information in the area below, taken from `http://<IP>/api/system`

```json
  Paste System information here....


```

### TO REPRODUCE

connect a mx400 device to your system.

### EXPECTED BEHAVIOUR

MX400 is discovered

### SCREENSHOTS

_If applicable, add screenshots to help explain your issu

### ADDITIONAL CONTEXT

_Add any other context about the issue here._

**(Please remember to close the issue when it has been addressed)**

**Comment by MichaelDvP** [comment]
> "type":"generic","name":"MX400/unknown","deviceID":"0x4B","productID":17,

Yes both new devices are the MX400 (@proddy : nice to have the telegram 01 name here), don't know  whyit uses two device ids.
I'll add to database,so it will show up as gateway. But gateways does not have any own entiy, so you miss nothing meanwhile.
